snap/snapcraft.yaml

```diff
@@ -29,6 +29,8 @@ package-repositories:
     ppa: data-platform/mysqld-exporter
   - type: apt
     ppa: data-platform/mysqlrouter-exporter
+  - type: apt
+    ppa: data-platform/mysql-archive # needed for libprotobuf32
```

Comment:
This must NOT be committed, It is a recycle bin. You will have mess in packages.

Reply:
I have restored libprotobuf32 into mysql-shell PPA, it used to use from xtrabackup PPA but was removed there as no longer necessary. Should be solved now: https://launchpad.net/~data-platform/+archive/ubuntu/mysql-shell/+packages

Reply:
Removed mysql-archive PPA from list of repositories in 08052dbdf1f797d68762fac8505316ed508bacde